apps/web/components/Organization/Wrapper/Themes/RettindagaeslaFatladsFolksTheme/RettindagaeslaFatladsFolksHeader.tsx

```diff
@@ -12,9 +16,23 @@ interface Props {
 
 const RettindagaeslaFatladsFolksHeader = ({ organizationPage }: Props) => {
   const { linkResolver } = useLinkResolver()
+  const namespace = useMemo(
+    () => JSON.parse(organizationPage.organization?.namespace?.fields ?? '{}'),
+    [organizationPage.organization?.namespace?.fields],
+  )
+  const n = useNamespace(namespace)
+
   const { width } = useWindowSize()
   const isMobileScreenWidth = width < theme.breakpoints.lg
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'